sstables_loader.cc

```diff
@@ -499,9 +519,35 @@ future<> sstables_loader::download_task_impl::run() {
     llog.debug("Loading sstables from {}({}/{})", _endpoint, _bucket, _prefix);
     auto [ table_id, sstables_on_shards ] = co_await replica::distributed_loader::get_sstables_from_object_store(_loader.local()._db, _ks, _cf, _sstables, _endpoint, _bucket, _prefix, cfg);
     llog.debug("Streaming sstables from {}({}/{})", _endpoint, _bucket, _prefix);
-    co_await _loader.invoke_on_all([this, &sstables_on_shards, table_id] (sstables_loader& loader) mutable -> future<> {
-        co_await loader.load_and_stream(_ks, _cf, table_id, std::move(sstables_on_shards[this_shard_id()]), false, false);
-    });
+    std::exception_ptr ex;
+    try {
+        _as.check();
+        co_await _loader.invoke_on_all([this, &sstables_on_shards, table_id] (sstables_loader& loader) mutable -> future<> {
+            co_await loader.load_and_stream(_ks, _cf, table_id, std::move(sstables_on_shards[this_shard_id()]), false, false);
```

Comment:
is load_and_stream abortable?
should it be?
it seems to me it might take a long time for very large sstables

Reply:
It will simply fail because a sstable read will fail, once the shard abort source causes a s3 client read to fail. 
I considered adding an additional explicit "cancel()" method to the streamer, but I don't think it is necessary.

Reply:
okay, makes sense